Codeherder19/flashcards-starter
# Feature/initial file creation and setup

### What does this PR do?
- This PR creates the necessary test.js files, as well as the other class files for Deck, Turn, and Round. Also created Turn and Card classes with methods thus far.

### What does it fix?
- No fix, just the beginning of a feature that will allow this flash card game to be played in the command line.

### Is this a feature or a fix?
- This will be a feature once it is complete.

### Where should the reviewer start?
- No review requested as of yet.

### How should this be tested?
- The game cannot actually be played until iteration 3, so unit tests will have to be designed to ensure things are written properly and will run correctly once we approach iteration 3.
